types/tz-lookup/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for tz-lookup 6.1
+// Project: https://github.com/darkskyapp/tz-lookup#readme
+// Definitions by: Ulf Winkelvos <https://github.com/uwinkelvos>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function tz_lookup(lat: number, long: number): string;
+declare namespace tz_lookup { }
```

Comment:
I think what you're looking for is `export as namespace tzlookup;` for the global scope? otherwise, is there a reason for an empty namespace?

Reply:
hey! I am sorry! that was just a leftover. Removed the namespace all together.